src/platform/tls_schannel.c

```diff
@@ -1177,12 +1171,10 @@ CxPlatTlsSecConfigCreate(
 
 
 #ifdef _KERNEL_MODE
-    if (IsClient) {
-        //
-        // Nothing supported for client right now.
-        //
-
-    } else if (CredConfig->Type == QUIC_CREDENTIAL_TYPE_CERTIFICATE_HASH) {
+    //
+    // Assumption: client works the same as server here
```

Comment:
Assumption? If this is a to-do, can you rephrase this in terms of a `TODO:` comment?